youtube_dl/extractor/cultureunplugged.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+
+class CultureUnpluggedIE(InfoExtractor):
+    _VALID_URL = r'http?://(?:www\.)?cultureunplugged\.com/documentary\/watch-online\/play\/(?P<id>\d+)\/(?P<display_id>[^/]+)'
```

Comment:
Valid point for the slashes. Still refining regex. It doesn't appear the server returns HTTPS, but no harm in adding it. 

I added the display ID as another optional ID, which mimics the title. Should this be extracted outside of the valid URL?